docs/guides/deploy-guide/bootstrap.md

```diff
@@ -114,41 +167,45 @@ the nodes are already ready for use.
      to host\r", "unreachable": true}
      ```
 
-7. Refresh facts.
+## Bootstrap the nodes
+
+1. Refresh facts.
```

Comment:
A newcomer may ask: What exactly is this step responsible for and what purpose does it serve?

I think it would be good if we could link to a documentary here that goes into this.